bundle/manifests/search-v2-operator.clusterserviceversion.yaml

```diff
@@ -0,0 +1,217 @@
+    operators.operatorframework.io/project_layout: go.kubebuilder.io/v3
+  name: search-v2-operator.v2.0.0
+  namespace: placeholder
+spec:
+  apiservicedefinitions: {}
+  customresourcedefinitions:
+    owned:
+    - description: OCMSearch is the Schema for the ocmsearches API
+      displayName: OCMSearch
+      kind: OCMSearch
+      name: ocmsearches.cache.open-cluster-management.io
+      version: v1
+  description: Operator to install ACM Search
+  displayName: search-v2-operator
+  icon:
+  - base64data: ""
+    mediatype: ""
+  install:
+    spec:
+      clusterPermissions:
+      - rules:
+        - apiGroups:
+          - cache.open-cluster-management.io
+          resources:
+          - ocmsearches
+          verbs:
+          - create
+          - delete
+          - get
+          - list
+          - patch
+          - update
+          - watch
+        - apiGroups:
+          - cache.open-cluster-management.io
+          resources:
+          - ocmsearches/finalizers
+          verbs:
+          - update
+        - apiGroups:
+          - cache.open-cluster-management.io
+          resources:
+          - ocmsearches/status
+          verbs:
+          - get
+          - patch
+          - update
+        - apiGroups:
+          - authentication.k8s.io
+          resources:
+          - tokenreviews
+          verbs:
+          - create
+        - apiGroups:
+          - authorization.k8s.io
+          resources:
+          - subjectaccessreviews
+          verbs:
+          - create
+        - apiGroups:
+          - ''
+          resources:
+          - secrets
+          - services
+          verbs:
+          - get
+          - list
+          - create
+          - update
+          - delete
+          - watch
+        - apiGroups:
+          - apps
+          resources:
+          - deployments
+          verbs:
+          - '*'          
+        serviceAccountName: search-v2-operator
+      deployments:
+      - name: search-v2-operator-controller-manager
+        spec:
+          replicas: 1
+          selector:
+            matchLabels:
+              control-plane: controller-manager
+          strategy: {}
+          template:
+            metadata:
+              labels:
+                control-plane: controller-manager
+            spec:
+              containers:
+              - args:
+                - --secure-listen-address=0.0.0.0:8443
+                - --upstream=http://127.0.0.1:8080/
+                - --logtostderr=true
+                - --v=10
+                image: gcr.io/kubebuilder/kube-rbac-proxy:v0.8.0
+                name: kube-rbac-proxy
+                ports:
+                - containerPort: 8443
+                  name: https
+                  protocol: TCP
+                resources:
+                  limits:
+                    cpu: 200m
+                    memory: 100Mi
+                  requests:
+                    cpu: 100m
+                    memory: 50Mi
```

Comment:
Requests seem high, especially the cpu.  I believe we have removed all cpu requests and limits from our pods.

Reply:
I will reduce , the request - but I want to have the request here - may be useful to debug problems